src/table/types.ts

```diff
@@ -218,6 +223,7 @@ export interface CellHOCProps extends TableCellProps {
   cell: Cell;
   column: Column;
   announce: Announce;
+  isFlagEnabled: (flag: string) => boolean;
```

Comment:
Line 152 and 192 has this typed in this shape, might be worth keep the same way.

```suggestion
  isFlagEnabled(flag: string): boolean;
```

Reply:
agree, will change